Description: Use Appstream current spec.
 Update appstream metadata info as per spec at
 https://wiki.debian.org/AppStream/Guidelines.
Forwarded: https://bugs.launchpad.net/yarock/+bug/1813638/
Author: Scarlett Moore <sgmoore@kde.org>
Last-Update: 1/31/19

Index: Yarock_1.3.1_Sources/data/org.yarock.appdata.xml
===================================================================
--- Yarock_1.3.1_Sources.orig/data/org.yarock.appdata.xml	2019-02-01 21:11:37.466968059 +0000
+++ Yarock_1.3.1_Sources/data/org.yarock.appdata.xml	2019-02-01 21:13:36.684260345 +0000
@@ -1,39 +1,50 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<application>
-  <id type="desktop">yarock.desktop</id>
-  <licence>CC0-1.0</licence>
-  <project_license>GPL-3.0</project_license>
-  <summary>Modern linux music player</summary>
-  <description>
-    <p>Yarock is a music player designed to provide a clean, simple and beautiful music collection based on album coverart.</p>
-    <p>You can easily browse and search your local music collection through various views such as artists, albums, tracks, genre, years, etc. You can also browse your play radio stream thinks to TuneIn, Shoutcast, Dirble radio services.</p>
-    <p>Features include.</p>
-    <ul>
-        <li>Music collection database (SQLite 3)</li>
-        <li>Browse your local music collection based on cover art</li>
-        <li>Easy search and filter music collection</li>
-        <li>Manage favorites item (album, artist)</li>
-        <li>Play music directly from collection or playqueue</li>
-        <li>Simple optionnal playqueue</li>
-        <li>Smart playlist generator</li>
-        <li>Support mp3,ogg vorbis,flac music files</li>
-        <li>Support load/save playlist file (m3u, pls, xspf)</li>
-        <li>Play radio stream (TuneIn, Shoutcast, Dirble)</li>
-        <li>Contextual info and cover art download (lastfm, echonest, musicbrainz)</li>
-        <li>Last Fm scrobbler support</li>
-        <li>Mp3Gain tag support for volume normalization</li>
-        <li>Command line interface, Mpris interface</li>
-        <li>Clean and simple user interface</li>
-        <li>No GNOME or KDE dependancies</li>
-    </ul>
-  </description>
-  <url type="homepage">http://qt-apps.org/content/show.php/yarock?content=129372</url>
-  <screenshots>
-    <screenshot type="default" width="1248" height="702">https://launchpadlibrarian.net/189537409/screenshot_album_view.png</screenshot>
-    <screenshot width="1248" height="702">https://launchpadlibrarian.net/189537418/screenshot_artist_view_1.png</screenshot>
-    <screenshot width="1248" height="702">https://launchpadlibrarian.net/189537419/screenshot_artist_view_2.png</screenshot>
-    <screenshot width="1248" height="702">https://launchpadlibrarian.net/189537431/screenshot_track_view.png</screenshot>
-    <screenshot width="1248" height="702">https://launchpadlibrarian.net/189537441/screenshot_tunin_view.png</screenshot>
-  </screenshots>
-  <updatecontact>updatecontact@yarock.org</updatecontact>
-</application>
+<component type="desktop">
+ <id>org.yarock.desktop</id>
+ <name>Yarock</name>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <summary>Modern linux music player</summary>
+ <description>
+  <p>Yarock is a music player designed to provide a clean, simple and beautiful music collection based on album coverart.</p>
+  <p>You can easily browse and search your local music collection through various views such as artists, albums, tracks, genre, years, etc. You can also browse your play radio stream thinks to TuneIn, Shoutcast, Dirble radio services.</p>
+  <p>Features include.</p>
+  <ul>
+   <li>Music collection database (SQLite 3)</li>
+   <li>Browse your local music collection based on cover art</li>
+   <li>Easy search and filter music collection</li>
+   <li>Manage favorites item (album, artist)</li>
+   <li>Play music directly from collection or playqueue</li>
+   <li>Simple optionnal playqueue</li>
+   <li>Smart playlist generator</li>
+   <li>Support mp3,ogg vorbis,flac music files</li>
+   <li>Support load/save playlist file (m3u, pls, xspf)</li>
+   <li>Play radio stream (TuneIn, Shoutcast, Dirble)</li>
+   <li>Contextual info and cover art download (lastfm, echonest, musicbrainz)</li>
+   <li>Last Fm scrobbler support</li>
+   <li>Mp3Gain tag support for volume normalization</li>
+   <li>Command line interface, Mpris interface</li>
+   <li>Clean and simple user interface</li>
+   <li>No GNOME or KDE dependancies</li>
+  </ul>
+ </description>
+ <url type="homepage">https://qt-apps.org/content/show.php/yarock?content=129372</url>
+ <screenshots>
+  <screenshot type="default">
+   <image type="source" width="1248" height="702">https://launchpadlibrarian.net/189537409/screenshot_album_view.png</image>
+  </screenshot>
+  <screenshot>
+   <image type="source" width="1248" height="702">https://launchpadlibrarian.net/189537418/screenshot_artist_view_1.png</image>
+  </screenshot>
+  <screenshot>
+   <image type="source" width="1248" height="702">https://launchpadlibrarian.net/189537419/screenshot_artist_view_2.png</image>
+  </screenshot>
+  <screenshot>
+   <image type="source" width="1248" height="702">https://launchpadlibrarian.net/189537431/screenshot_track_view.png</image>
+  </screenshot>
+  <screenshot>
+   <image type="source" width="1248" height="702">https://launchpadlibrarian.net/189537441/screenshot_tunin_view.png</image>
+  </screenshot>
+ </screenshots>
+ <update_contact>updatecontact@yarock.org</update_contact>
+</component>
